lightning/Cargo.toml

```diff
@@ -32,6 +32,8 @@ unsafe_revoked_tx_signing = []
 no-std = ["hashbrown", "possiblyrandom", "libm"]
 std = []
 
+dnssec = ["dnssec-prover/validation"]
```

Comment:
thinking if we can make the `dnssec-prover` dependencies optional and included just when this feature is specified?

Reply:
I see, and I agree. I was thinking that maybe the `dnssec` will be a popular feature that we want enabled most of the time